### DISCUSSION TITLE
[BUG]: 500 Server Error When Accessing OSCR from Feed Page

### DISCUSSION THREAD

**Discussion by a-ridley** [open-sauced/feedback#38] [General feedback or Bugs]
### What am I doing & What did I expect?
I am trying to view my OSCR from my feed page. When I click on the 'See it now' button on my feed page I get an application server error. I expect this to take me to my dev card and show my rating.

### Steps to reproduce
1. Go to https://app.opensauced.pizza/feed
2. Scroll down to find the 'Introducing the OSCR!' banner. [see attatched img]
<img width="291" alt="image" src="https://github.com/user-attachments/assets/7c48e256-3231-45cd-9378-273cf6798286">
3. Click 'See it now!'
4. Experience a 500 error
<img width="1435" alt="image" src="https://github.com/user-attachments/assets/d0bed67e-f968-47dc-bba6-2e1013b2a234">

**Comment by jpmcb** [comment]
Created an issue: https://github.com/open-sauced/community/issues/39

Thanks for raising this issue!!
